src/index.ts

```diff
@@ -156,6 +161,12 @@ export interface InvokeRightRequest {
   jurisdictionCode: string
   rightCode: string
   user: User
+
+  // dataSubjectTypeCode is the identifier representing the data subject type specified by the user
+  dataSubjectTypeCode: string
+
+  // dataSubjectTypeUserInput is additional information provided by the user describing their relation to the business
+  dataSubjectTypeUserInput: string
```

Comment:
Why is this not in `User` (which we should rename to `DataSubject`)?

Reply:
ah oops it should be

Reply:
fixed